pkg/loop/internal/pb/chain_reader.proto

```diff
@@ -12,14 +12,14 @@ service ChainReader {
   rpc BatchGetLatestValues (BatchGetLatestValuesRequest) returns (BatchGetLatestValuesReply) {}
   rpc QueryKey(QueryKeyRequest) returns (QueryKeyReply) {}
   rpc Bind(BindRequest) returns (google.protobuf.Empty) {}
+  rpc Unbind(UnbindRequest) returns (google.protobuf.Empty) {}
 }
 
 // GetLatestValueRequest has arguments for [github.com/smartcontractkit/chainlink-common/pkg/types.ChainReader.GetLatestValue].
 message GetLatestValueRequest {
-  string contract_name = 1;
-  string method = 2;
-  Confidence confidence = 3;
-  VersionedBytes params = 4;
+  string read_name = 1;
```

Comment:
read_name is used in some places to signify a specific contact read (method or event on evm). Maybe read_identifier?